cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CruiseControlConfiguration.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.strimzi.operator.cluster.model;
+
+import io.strimzi.api.kafka.model.CruiseControlSpec;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static java.util.Arrays.asList;
+import static java.util.Collections.emptyList;
+
+/**
+ * Class for handling Cruise Control configuration passed by the user
+ */
+public class CruiseControlConfiguration extends AbstractConfiguration {
+
+    /**
+     * A list of case insensitive goals that Cruise Control supports in the order of priority.
+     * The high priority goals will be executed first.
+     */
+    private static final String DEFAULT_GOALS = "com.linkedin.kafka.cruisecontrol.analyzer.goals.RackAwareGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.ReplicaCapacityGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.DiskCapacityGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.NetworkInboundCapacityGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.NetworkOutboundCapacityGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.CpuCapacityGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.ReplicaDistributionGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.PotentialNwOutGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.DiskUsageDistributionGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.NetworkInboundUsageDistributionGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.NetworkOutboundUsageDistributionGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.CpuUsageDistributionGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.TopicReplicaDistributionGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.LeaderReplicaDistributionGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.LeaderBytesInDistributionGoal";
+
+    private static final String DEFAULT_GOAL_PRIORITY = DEFAULT_GOALS;
```

Comment:
Do you even need a separate EnvVar here? The user will override the priority if they want so there is no need to a seperate default value?